app/controllers/concerns/rendering.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Rendering logic for all controllers
+module Rendering
+  extend ActiveSupport::Concern
+
+  included do
+    def render_json(model, **args)
+      opts = ['index'].include?(action_name) ? metadata : {}
+      opts.merge!(include: params[:include].split(',')) if params[:include]
+      render({ json: serializer.new(model, opts) }.merge(args))
+    end
+
+    def render_error(model, **args)
```

Comment:
Is this used anywhere?

Reply:
Good catch - this one was from create/update, but we'll be using it soon :)